app/scripts/modules/oracle/pipeline/stages/destroyAsg/destroyAsgStage.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const angular = require('angular');
+
+import {
+  StageConstants
+} from '@spinnaker/core';
+
+module.exports = angular.module('spinnaker.oraclebmcs.pipeline.stage.destroyAsgStage', [])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'destroyServerGroup',
+      cloudProvider: 'oraclebmcs',
+      templateUrl: require('./destroyAsgStage.html'),
+      executionDetailsUrl: require('core/pipeline/config/stages/destroyAsg/templates/destroyAsgExecutionDetails.template.html'),
```

Comment:
Oh, this should now be `PipelineTemplates.destroyAsgExecutionDetails` (add `PipelineTemplates` to the import above from `@spinnaker/core`)